Projects/Playground/Playground.Wpf/App.xaml.cs

```diff
@@ -4,7 +4,7 @@
 
 namespace Playground.Wpf
 {
-    public partial class App : MvxApplication<MvxWpfSetup<Core.App>, Core.App>
+    public partial class App : MvxApplication
```

Comment:
Can this not inherit from the generic base?

Reply:
No i couldn't get the generic type arguments to work on WPF.